### PR TITLE
Normalize tests for expected errors in samples

### PR DESCRIPTION
**Pull request by pushkine**
On top of #868
Comparison: https://github.com/pushkine/language-tools/compare/share-sample-logic...pushkine:normalize-test-errors

### CODE REVIEW

File: packages/svelte2tsx/test/htmlx2jsx/index.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import { htmlx2jsx } from '../build/htmlxtojsx';
 import { test_samples } from '../helpers';
 
-describe('htmlx2jsx', () => {
-    test_samples(__dirname, htmlx2jsx, 'jsx');
+describe('htmlx2jsx', function () {
```
Comment (dummdidumm) [inline packages/svelte2tsx/test/htmlx2jsx/index.ts:4]: why the change to `function` (here and below)?

File: packages/svelte2tsx/test/svelte2tsx/index.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
 import svelte2tsx from '../build/index';
 import { test_samples } from '../helpers';
 
-describe('svelte2tsx', () => {
+describe('svelte2tsx', function () {
```
Comment (dummdidumm) [inline packages/svelte2tsx/test/svelte2tsx/index.ts:4]: Why the change to `function` (here and below)?

**Review by dummdidumm — COMMENTED** [review]
Looks good to me. Could you fix the merge conflicts?

File: packages/svelte2tsx/test/htmlx2jsx/index.ts (hunk 1 above)
Comment (pushkine) [reply packages/svelte2tsx/test/htmlx2jsx/index.ts]: I took the habit to avoid inline functions when possible (worse performance), I should avoid doing that in PRs

**Review by dummdidumm — APPROVED** [review]
Thanks!
